### PR TITLE
feat: add abi json for verified contracts

### PR DESCRIPTION
**Pull request by MexicanAce**
# What ❔

Display the contract's ABI JSON if it's verified

## Why ❔

This addresses #242 and allows users to easily copy/paste the ABI for easier development

## Checklist

- [x] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.

## Evidence

Verified contract with a small ABI:
image

Verified contract with a large ABI (Closed & Expanded):
image
image

Unverified contract:
image

### CODE REVIEW

File: packages/app/src/locales/uk.json [hunk 1]
```diff
@@ -215,6 +215,9 @@
             "verifyButton": "Завірити Смарт Контракт",
             "deployedBytecode": "Розгорнутий байт-код",
             "sourceCode": "Вихідний код"
+        },
+        "abiInteraction": {
+            "contractAbi": "ABI Контракт"
```
Comment (Romsters) [inline packages/app/src/locales/uk.json:220]:
```suggestion
            "contractAbi": "ABI контракту"
```

File: packages/app/src/components/common/table/fields/AbiData.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+const props = defineProps({
+  value: {
+    type: String,
+    default: "",
+    required: true,
+  },
+});
+
+const truncatedAbi = computed<string>(() => {
+  return props.value.replace(/(.{600})..+/, "$1...");
```
Comment (Romsters) [inline packages/app/src/components/common/table/fields/AbiData.vue:43]: What is this for?

**Comment by JackHamer09** [comment]
Do you think it would make sense to keep it in pretty format? `JSON.stringify(abi, null, 2)`
This way you can search for specific piece of abi, e.g. specific function

**Review by Romsters — APPROVED** [review]

**Comment by MexicanAce** [comment]
> Do you think it would make sense to keep it in pretty format, for readability? JSON.stringify(abi, null, 2)
This way you can search for specific piece of abi, e.g. specific function

We decided to keep this as close to Etherscan's implementation for a better UX

**Review by Romsters — APPROVED** [review]

**Pull request merged**